### ISSUE TITLE
Item duplication on reset.

### ISSUE THREAD

**Issue by Hiveul** [tewtal/alttp_sm_combo_randomizer#45]
Console or Emulator: SNES on SD2SNES

 If emulator state which emulator and version.

Original Filename: SMALttP - valttp-no-glitches-v9.2_normal-open-ganon_combo_977881435

Seed: 977881435

Description of problem:

This seed generated me 2 Energy Tanks in the Dark Palace. However, the key logic confused me and I reset the game without saving for fear of having missed some items. One time I used the console reset (a short one that didn't boot on sd2snes menu) and the other time I used the reset button combination. Both times I thought the game had saved my LttP progress because my energy tanks had increased in SM, however the palace was still pristine and unexplored. I haven't progressed my game enough to know what will happen if I end up finding more than the max number of tanks as a result.

**Comment by Hiveul** [comment]
I have finished my runthrough and I did end up with 3 more Energy Tanks than I was supposed to. They would function normally even though they wouldn't display ingame. They did, however, display in a mildly glitch fashion on the load game screen.

I'm aware this is likely due to some stuff needing to be written to srm in order to make the two games work and I don't really expect this to be fixable, but I think the casual player might like to know that resetting might cause accidental cheating.
